armi/operators/operator.py

```diff
@@ -621,22 +608,12 @@ def interactAllEveryNode(self, cycle, tn, excludedInterfaceNames=None):
         excludedInterfaceNames : list, optional
             Names of interface names that will not be interacted with.
         """
-        excludedInterfaceNames = excludedInterfaceNames or ()
-        activeInterfaces = [
-            ii
-            for ii in self.interfaces
-            if ii.enabled() and ii.name not in excludedInterfaceNames
-        ]
+        activeInterfaces = self.getActiveInterfaces("EveryNode")
         self._interactAll("EveryNode", activeInterfaces, cycle, tn)
 
     def interactAllEOC(self, cycle, excludedInterfaceNames=None):
         """Interact end of cycle for all enabled interfaces."""
-        excludedInterfaceNames = excludedInterfaceNames or ()
-        activeInterfaces = [
-            ii
-            for ii in self.interfaces
-            if ii.enabled() and ii.name not in excludedInterfaceNames
-        ]
+        activeInterfaces = self.getActiveInterfaces("EOC")
```

Comment:
It appears that you've lost the `excludedInterfaceNames` in a number of these calls.

Reply:
Okay, let me do a tally of the `excludedInterfaceNames` logic, just to be sure:

* `interactAllBOL` - uses `excludedInterfaceNames` and `self.cs["deferredInterfaceNames"]`
* `interactAllBOC` - uses `self.cs["deferredInterfacesCycle"]`, if the cycle is early enough
* `interactAllEveryNode` - uses `excludedInterfaceNames`
* `interactAllEOC` - uses `excludedInterfaceNames`
* `interactAllEOL` - uses neither
* `interactAllCoupled` - uses neither

Reply:
@keckler and @albeanth  It looks to me like the new logic (in the new `getActiveInterfaces()` method matches the old logic.

Feel free to fact-check me on that.

Reply:
Okay, I see it.

1 minute

Reply:
Fixed